src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -124,13 +148,30 @@ void* LoadCoreClr(std::string& runtimeDirectory)
         runtimeDirectory.erase(lastSlash);
```

Comment:
I don't know what happens in the code after this function exits but since `runtimeDirectory` is passed by reference this line changes the `runtimeDirectory` that is used by upstream callers. Is it intentional?